src/uct/ib/base/ib_iface.c

```diff
@@ -1568,6 +1572,7 @@ UCS_CLASS_INIT_FUNC(uct_ib_iface_t, uct_iface_ops_t *tl_ops,
     if (uct_ib_iface_is_roce(self) || config->is_global ||
         uct_ib_grh_required(uct_ib_iface_port_attr(self)) ||
         uct_ib_address_gid_is_global(&self->gid_info.gid) ||
+        (uct_ib_iface_port_attr(self)->lid == 0) ||
```

Comment:
is it some efa specific?

Reply:
yes seems EFA do not have LID, only using GID.